sdk/python/scripts/test_fast.sh

```diff
@@ -4,20 +4,20 @@ PULUMI_TEST_COVERAGE_PATH=$PULUMI_TEST_COVERAGE_PATH
 
 set -euo pipefail
 
-# TODO the ignored test seems to fail in pytest but not unittest. Need
-# to trackdown why.
+mkdir -p ../../junit
+JUNIT_DIR=$(realpath ../../junit)
 
-coverage run --append -m pytest lib/test \
+coverage run --append -m pytest --junitxml "$JUNIT_DIR/python-test-fast-a.xml" lib/test \
              --ignore lib/test/automation \
              --ignore lib/test/langhost/resource_thens/test_resource_thens.py
 
-coverage run --append -m unittest \
+coverage run --append -m pytest --junitxml "$JUNIT_DIR/python-test-fast-b.xml" \
```

Comment:
If it works fine in pytest, maybe this can be folded up into the run above, and then we don't have to have `-a.xml` and `-b.xml`? (could be done in a separate change)

Reply:
Oh yeah, that's a good idea!